doc/md_src_files/100_limitations.md

```diff
@@ -2,12 +2,18 @@
 
 # Limitations
 
-Even as a working proposal, SSVC has some limitations. These are inherent limits of the approach, which should be understood as tradeoffs. There are other limiting aspects of our implementation, but those have been covered as topics that need improvement and are described in [Future Work](#future-work).
+SSVC has some inherent limits in its approach, which should be understood as tradeoffs.
+There are other limiting aspects of our implementation, but those have been covered as topics that need improvement and are described in [Future Work](#future-work).
 
-We made two important tradeoffs compared to the current state of the practice with CVSS:
+We made two important tradeoffs compared to the current state of the practice.
+Other systems make different tradeoffs, which may be better or worse depending on the context of use.
+While these are inherently limitations of SSVC, we do not intend SSVC to be the one and only vulnerability management tool available.
+Those for whom these limitations are a must-have may be better supported by a different vulnerability management framework.
 
 1.  We eliminated numerical scores; this may make some practitioners uncomfortable. We explained the reasons for this in depth, but even though CVSS contains false precision, we still must contend with the fact that, psychologically, users find that comforting. As this comfort gap may negatively impact adoption, this fact is a limitation. Although it is ungainly, it would be sound to convert the priority outcomes to numbers at the end of the process, if existing processes require it. Which numbers we choose to convert to is immaterial, as long as the ordering is preserved. CVSS has set a precedent that higher numbers are worse, so a scale \[1, 2, 3, 4\] would work, with defer = 1 and immediate = 4. However, if it were important to maintain backwards compatibility to the CVSS range zero to ten, we could just as well relabel outcomes as \[2, 5.5, 8, 9.5\] for the midpoints of the current CVSS severity ranges.
 
```

Comment:
If we're going to suggest a numeric range, then it should state that there's no "calculation". (unless there would be!) Manipulating numbers allows for 'rounding errors' or approximations that some companies would like to have available.